test/unit/prbuf.c

```diff
@@ -41,37 +57,123 @@ payload_large_init(void)
 		payload_large[i] = (char)i;
 }
 
+/** Just some offset to test reader with. */
+#define BUFFER_OFFSET 33
+
+static inline const char *
+diagmsg(void)
+{
+	return diag_last_error(diag_get())->errmsg;
+}
+
+static int
+save_buffer(void *buf, int size)
+{
+	int fd = open("prbuf.bin", O_CREAT | O_TRUNC | O_RDWR, 0644);
+	if (fd == -1)
+		fail("open(prbuf.bin)", "== -1");
```

Comment:
Let's `unlink` the file right here so as not to pollute the test directory.

Reply:
```diff
--- a/test/unit/prbuf.c
+++ b/test/unit/prbuf.c
@@ -70,6 +70,7 @@ static int
 save_buffer(void *buf, int size)
 {
        int fd = open("prbuf.bin", O_CREAT | O_TRUNC | O_RDWR, 0644);
+       unlink("prbuf.bin");
        if (fd == -1)
                fail("open(prbuf.bin)", "== -1");
```